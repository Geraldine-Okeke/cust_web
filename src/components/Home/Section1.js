import './Home.css'
import { Link } from 'react-router-dom';
export default function Section1() {
  

  return (
    <div className='Top-main w-full '>
      <div className='md:float-right  justify-center items-center  w-full mt-20 md:w-full lg:w-1/2 xl:w-1/2 text-white px-3  md:p-20 flex flex-col'>
        <h1 className='mb-10 text-4xl text-center text-yellow-400 font-bold'>GATE <br/> FOUNDATION</h1>
        <p>
          Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Maecenas porttitor congue massa. 
          Fusce posuere, magna sed pulvinar ultricies, purus lectus malesuada libero,
          sit amet commodo magna eros quis urna. Nunc viverra imperdiet enim. Fusce est. 
        </p>
        <Link to="/GET_INVOLVED/Get_Involved">
          <button className=' bg-blue-900  text-yellow-400 rounded-full h-12 mt-10 mb-10 md:px-32 sm:px-16 padding font-bold'>GET INVOLVED</button>
        </Link>
        <Link>
          <button className=' bg-blue-900 text-yellow-400  rounded-full h-12 font-bold mb-10 md:px-32 sm:px-10 padding whitespace-nowrap '>CAUSE SHOP</button>
        </Link>
       
      </div>
    </div>
      
    
  );
}
