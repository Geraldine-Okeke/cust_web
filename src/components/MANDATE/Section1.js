import React from 'react';
import './Mandate.css';

export default function Section1(props) {
  const myStyle = {
    backgroundImage: `url(${props.bg})`,
    backgroundSize: 'cover',
    backgroundRepeat: 'no-repeat',
    width: '100%',
    height: 'auto'
  };

  const containerClassName = `float-${props.floatDirection || 'left'} md:pl-10 my-52 text-white text-center md:w-1/2 w-full p-4`;

  return (
    <div className="" style={myStyle}>
      <div className={containerClassName} >
        <h1 className='text-4xl font-bold '>{props.title}</h1>
        <p className='mt-20 text-start leading-8'>{props.details}</p>
      </div>
    </div>
  );
}
