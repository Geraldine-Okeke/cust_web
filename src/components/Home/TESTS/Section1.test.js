import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom'; 
import Section1 from '../Section1'; 

test('Section1 component snapshot', () => {
  const component = renderer.create(
    <MemoryRouter>
      <Section1 />
    </MemoryRouter>
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
