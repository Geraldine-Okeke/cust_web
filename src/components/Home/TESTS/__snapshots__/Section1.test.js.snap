// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section1 component snapshot 1`] = `
<div
  className="Top-main w-full "
>
  <div
    className="md:float-right w-full mt-20 md:w-full lg:w-1/2 xl:w-1/2 text-white px-3  md:p-20 flex flex-col"
  >
    <h1
      className="mb-10 text-2xl text-center font-bold"
    >
      KINDLEGATE 
      <br />
       FOUNDATION
    </h1>
    <p>
      Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Maecenas porttitor congue massa. Fusce posuere, magna sed pulvinar ultricies, purus lectus malesuada libero, sit amet commodo magna eros quis urna. Nunc viverra imperdiet enim. Fusce est.
    </p>
    <a
      href="/GET_INVOLVED/Get_Involved"
      onClick={[Function]}
    >
      <button
        className=" blue text-black rounded-full h-12 mt-10 mb-10 w-full font-bold"
      >
        GET INVOLVED
      </button>
    </a>
    <button
      className=" blue text-black rounded-full h-12 font-bold mb-10  "
    >
      KINDLEGATE CAUSE SHOP
    </button>
  </div>
</div>
`;
